src/ephy-sync-crypto.c

```diff
@@ -0,0 +1,952 @@
+/* -*- Mode: C; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
```

Comment:
OK, going to stop here for now... not looking forward to reviewing this file, since I've never understood the authentication you're doing with the sync server....